frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/selectors/immutable/wizard-selectors.ts

```diff
@@ -55,12 +60,39 @@ export const getStepsMetadata = (state, wizardID: string): VMWizardTabsMetadata
     delete result[tab].value;
   });
 
+  const isProviderImport = iGetCommonData(state, wizardID, VMWizardProps.isProviderImport);
+  const isSimpleView = iGetCommonData(state, wizardID, VMWizardProps.isSimpleView);
+
+  const visibleTabls = ALL_VM_WIZARD_TABS.filter((tab) => !result[tab].isHidden);
+  const canJumpToAll =
+    result[VMWizardTab.RESULT].isLocked ||
+    result[VMWizardTab.RESULT].isValid ||
+    result[VMWizardTab.RESULT].isPending;
+  const isLocked = _.some(visibleTabls, (tab) => result[tab]?.isLocked);
+  const lastTabErrorFatal = isLastStepErrorFatal(state, wizardID);
+
+  // Can always jump to first tab
+  result[visibleTabls[0]].canJumpTo = true;
```

Comment:
this should depend on `!disableJumpToAll && !isLocked && !lastTabErrorFatal`

Reply:
try to go to a disk modal and you will see the difference in css of left bar

Reply:
fixed